packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/apis/helpers/migration.ts

```diff
@@ -28,15 +28,24 @@ export class MigrationHelper {
    * before storing it in the index. It will therefore throw if the document is in a higher / unknown version.
    */
   migrateInputDocument(document: SavedObjectUnsanitizedDoc): SavedObjectUnsanitizedDoc {
-    return this.migrator.migrateDocument(document, { allowDowngrade: false });
+    return this.migrator.migrateDocument(document, { allowDowngrade: false }); // if false (as by default), allows documents to be indeed without a downgrade option
```

Comment:
NIT: maybe it's me, but I'm not sure to get the meaning of your comment?

Reply:
That's a typo error: autocomplete in auto-incorrect mode 😆 
The comment isn't needed, I'll remove it completely.
```suggestion
    return this.migrator.migrateDocument(document, { allowDowngrade: false });
```